heroku/cli
# refactor(pg-v5): Move command `pg:maintenance` to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bD6OlYAK/view

This converts `pg:maintenance` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:maintenance -a APP` with an app that has an upcoming maintenance scheduled
3) Verify the command succeeds, displaying info about the maintenance
4) Compare with prod
